lessons/identity-property/langs/en-us/index.adoc

```diff
@@ -0,0 +1,109 @@
+
+[.lesson-instruction]
+- Now, turn to Discover the Identity Property. Look at the first problem. Translate the expressions in words into Circles of Evaluation. What do you Notice? What do you Wonder?
+** _Possible responses: All three Circles include @math{12 + 4}. All three Circles evaluate to @math{16}._
+- Complete the worksheet by filling in the blanks for the remaining Circles. Avoid putting the same operator more than once.
+- Keisha says that when she multiplies a number by zero, she always gets that number. Do you agree?
+** _No, multiplying by zero produces zero._
+- What did you observe about the Identity Property?
+** _Possible responses: I can multiply or divide by @math{1} and get an equivalent result. I can add or subtract @math{0} and get an equivalent result._
+
+=== Investigate
+
+[.lesson-instruction]
+- Analyze the Circles to decide Which One Doesn't Belong?
+- Be sure to explain why each Circle you select does not belong with the others.
+
+As students explain why various Circles do not belong with the others, encourage them to practice using vocabulary that they have encountered already. For instance, @math{12} is _not_ equal to @math{0 - 12} because the Commutative Property only holds for addition. Although this example illustrates misuse of the Identity Propery (we can subtract zero from twelve, but not twelve from zero!), thinking about the Commutative Property can help students recognize what is going on.
+
+=== Synthesize
+
+- For which operations does the Identity Property hold?
+** _To apply the Identity Property, we add or subtract zero, or we multiply or divide by one._
+- Did you encounter any instances where the Identity Property was used in combination with another property?
+** _Yes! The Identity Property can be used in combination with the other properties we've discussed to create equivalent Circles._
+
+
+== More Circle Transformations with Variables @duration{25 minutes}
+
+=== Overview
+Students analyze Circle Transformations with variables to determine equivalency, this time integrating their knowledge of the Identity Property..
+
+=== Launch
+
+We can represent the Identity Property with variables, in the same way that we could represent the Associative Property and the Commutative Property with variables. Here's an example:
+
+[.embedded, cols="^.^3,^.^1,^.^3", grid="none", stripes="none" frame="none"]
+|===
+|@show{(coe '(+ 3 q))}	| @math{=} | @show{(coe '(* (+ 3 q) 1))}
+| @math{3 + q} 	| @math{=} | @math{(3 + q) \times 1}
+|===
+
+Or like this:
+
+[.embedded, cols="^.^3,^.^1,^.^3", grid="none", stripes="none" frame="none"]
+|===
+|@show{(coe '(+ 6 t))}	| @math{=} | @show{(coe '(+ 0 (+ 6 t)))}
+| @math{6 + t} 			| @math{=} | @math{(6 + t) + 0}
```

Comment:
Why use different numbers and letters here? I would use q and 3 in both, to really show how it's the associative property and the expressions are equivalent